client/src/app/my-projects/page.tsx

```diff
@@ -0,0 +1,367 @@
+            getValue() === "Conservation"
+              ? "border-sky-300 bg-blue-500/20 text-sky-blue-300"
+              : "border-mint-green-200 bg-green-500/20 text-mint-green-200",
+          )}
+        >
+          {getValue()}
+        </span>
+      </div>
+    ),
+  },
+  {
+    accessorKey: "actions",
+    header: ActionsDropdown,
+    cell: ActionsDropdown,
+    className: "!border-l-0",
+    enableSorting: false,
+  },
+];
+
+const filters = [
+  { label: "All", count: 90 },
+  { label: "Conservation", count: 70 },
+  { label: "Restoration", count: 20 },
+];
+
+export default function MyProjects() {
+  const { open: navOpen } = useSidebar();
+  const [sorting, setSorting] = useState<SortingState>([
+    {
+      id: "projectName",
+      desc: false,
+    },
+  ]);
+  const [pagination, setPagination] = useState<PaginationState>({
+    pageIndex: 0,
+    pageSize: 10,
+  });
+  const [activeFilter, setActiveFilter] = useState("All");
+
+  const table = useReactTable({
+    data: MOCK_DATA.filter((project) =>
+      activeFilter === "All" ? true : project.type === activeFilter,
+    ),
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    manualPagination: true,
+    state: {
+      sorting,
+      pagination,
+    },
+    onSortingChange: setSorting,
+    onPaginationChange: setPagination,
+    enableRowSelection: true,
+    enableMultiRowSelection: true,
+  });
+
+  return (
+    <motion.div
+      layout
+      layoutDependency={navOpen}
+      className="flex flex-1 flex-col"
+      transition={LAYOUT_TRANSITIONS}
+    >
+      <div className="mx-3 flex h-full flex-1 flex-col">
+        <div className="flex items-center justify-between py-3">
+          <div className="flex items-center space-x-2">
+            <SidebarTrigger />
+            <h1 className="text-2xl font-medium">My projects</h1>
+          </div>
+          <Button variant="default">
+            <PlusCircledIcon className="mr-1 h-4 w-4" />
+            Project
+          </Button>
+        </div>
+
+        <div className="flex items-center justify-between py-4">
+          <div className="flex gap-2 rounded-full bg-slate-900">
+            {filters.map((filter) => (
+              <Button
+                key={filter.label}
+                variant="ghost"
+                onClick={() => setActiveFilter(filter.label)}
+                className={`rounded-full text-sm text-big-stone-200 ${activeFilter === filter.label ? "bg-big-stone-900 text-big-stone-50" : "hover:bg-big-stone-900"} `}
+              >
+                {filter.label} ({filter.count})
+              </Button>
+            ))}
+          </div>
+          <div className="flex items-center gap-2">
+            <Search className="h-4 w-4 text-muted-foreground" />
+            <input
+              type="search"
+              placeholder="Search"
+              className="bg-transparent outline-none"
+            />
```

Comment:
There's already a component for this: `Search`, you can see the implementation in the overview page here: https://github.com/Vizzuality/tnc-blue-carbon-cost-tool/blob/dev/client/src/containers/projects/table/toolbar/search/index.tsx#L17-L22